README.md

```diff
@@ -630,6 +632,10 @@ export SEALED_SECRETS_CONTROLLER_NAMESPACE=sealed-secrets
 kubeseal <mysecret.json >mysealedsecret.json
 ```
 
+### How to use one controller for a subset of namespaces
+
+In case you are using a multi tenant kubernetes cluster (shared cluster), and you're lacking clusterwide permissions to run a controller, but you want to use one controller for more than one namespace, you can provide all additional namespaces with the command line flags "all-namespaces=false" and "add-namespaces=<namespace1>,<namespace2>,<...>". Make sure you provide appropriate roles and rolebindings in the target namespaces, so the controller can manage the secrets in there.
```

Comment:
```suggestion
In case you are using a multi tenant Kubernetes cluster (shared cluster) and you're lacking clusterwide permissions to run a controller, but you want to use one controller for more than one namespace, you can provide all additional namespaces with the command line flag `--additional-namespaces=<namespace1>,<namespace2>,<...>`. Make sure you provide appropriate roles and rolebindings in the target namespaces, so the controller can manage the secrets in there.
```